app/views/orders/index.html.erb

```diff
@@ -18,7 +18,7 @@
 
       <tbody>
         <% @orders.each do |order| %>
-          <tr class="order" data-order-id="<%= order.id %>" data-href="<%= edit_order_path(order, redirect_to: 'orders') %>">
+          <tr class="order <%= bold_pending_order(order) %>" data-order-id="<%= order.id %>" data-href="<%= edit_order_path(order, redirect_to: 'orders') %>">
```

Comment:
Since you've moved the logic into Order you can call 'status_based_class` on the `order` object that each row has from the each loop like this:
```
<tr class="order <%= order.status_based_class %>" data-...
```